src/ARnft.js

```diff
@@ -87,6 +90,12 @@ export default class ARnft {
     return await nft._initialize(markerUrl, stats)
   }
 
+  renderer () {
```

Comment:
get the renderer object from the Event stream and return it to the getter. Probably better re-name this to getRenderer...